docs.html

```diff
@@ -994,8 +998,127 @@ <h2 class="fs-m">JavaScript</h2>
 
                 </script>
```

Comment:
suggestion:- useless spaces here

Reply:
I gave them to distinguish between code snippets and the actual actual